src/VisualStudio/Core/Def/TaskList/ExternalErrorDiagnosticUpdateSource.cs

```diff
@@ -487,7 +494,7 @@ private InProgressState GetOrCreateInProgressState()
                 // We take current snapshot of solution when the state is first created. and through out this code, we use this snapshot.
                 // Since we have no idea what actual snapshot of solution the out of proc build has picked up, it doesn't remove the race we can have
                 // between build and diagnostic service, but this at least make us to consistent inside of our code.
-                _stateDoNotAccessDirectly = new InProgressState(this, _workspace.CurrentSolution, _activeCancellationSeriesDoNotAccessDirectly.CreateNext(_disposalToken));
+                _stateDoNotAccessDirectly = new InProgressState(this, _workspace.CurrentSolution, _activeCancellationSeriesDoNotAccessDirectly.CreateNext());
```

Comment:
passing _disposalToken is not needed here.  When we created teh cancellation-series we pass our disposal token to it, so we always get this behavior when calling CreateNext.